share/xtp/xml/neighborlist.xml

```diff
@@ -1,12 +1,9 @@
 <options>
-
-
 <neighborlist help="Determines neighbouring pairs for exciton transport" section="sec:neighborlist">
         <segments help="cutoff distances for certain segments identified by name">
-                <type help="segment name1 segment_name2">DCV5T_ME3 DCV5T_ME3</type>
-                <cutoff unit="nm">3</cutoff>
+                <type help="segment name1 segment_name2" default="DCV5T_ME3 DCV5T_ME3">DCV5T_ME3 DCV5T_ME3</type>
```

Comment:
Having the whole `segments` option here makes no sense, because there are now really system specific names given. We should make this part 100% optional